Documentation/Contributing/development-flow.md

```diff
@@ -245,7 +245,13 @@ Common cases that may need tests:
 Rook's upstream continuous integration (CI) tests will run integration tests against your changes
 automatically.
 
-Integration tests will be run in Github actions. If an integration test fails, a tmate session will be available for troubleshooting for a short time. See the action details for an ssh connection to the Github runner.
+## Tmate Session
+
+Integration tests will be run in Github actions. If an integration test fails, enable a tmate session to troubleshoot the issue by one of the following steps:
+
+* Restart the CI action and click the "Enable debug logging" checkbox from the github UI, or
+* Add the label `debug-ci` to the PR and push your changes again.
+See the action details for an ssh connection to the Github runner.
```

Comment:
Need a blank line before this, to avoid this line being rendered as part of the last bullet.
```suggestion

See the action details for an ssh connection to the Github runner.
```